toolkit/tools/pkg/imagecustomizerlib/customizeos.go

```diff
@@ -75,12 +75,12 @@ func doOsCustomizations(buildDir string, baseConfigPath string, config *imagecus
 		return err
 	}
 
-	overlayUpdated, err := enableOverlays(config.OS.Overlays, imageChroot)
+	overlayUpdated, err := enableOverlays(config.OS.Overlays, config.OS.SELinux.Mode, config.Storage.FileSystems, imageChroot)
```

Comment:
Use the `selinuxMode` variable instead of `config.OS.SELinux.Mode`.

In addition, in `addEquivalencyRules` remove the use `bootCustomizer`.